src/analysis/ContextBar.test.ts

```diff
@@ -239,7 +240,7 @@ const cromwellOnAzureRunning: App = {
   region: 'us-central1',
 };
 
-const rstudioRuntime: Runtime = {
+const rstudioRuntime: ListRuntimeItem = {
```

Comment:
These could still be Runtime objects, right? Don't Runtimes now have the normalized region in their configs?

Reply:
(nit; not doing)